src/bindings/jna/CMakeLists.txt

```diff
@@ -57,24 +57,6 @@ if (Java_JAVAC_EXECUTABLE)
 				set (GRADLE_TEST_EXCLUDES "")
 				java_test_needs_plugin (GOptsTest gopts)
 
-				if ((CMAKE_SYSTEM_NAME STREQUAL "Darwin")
```

Comment:
How is this change related to the PR?

Reply:
Previously, all test have been explicitly added to not be executed on this mac os version. I took the opportunity to optimize the cmake file instead of adding more (new) test to the exclusion list.